artview/components/plot_radar.py

```diff
@@ -388,16 +385,18 @@ def NewRadar(self, variable, value, strong):
             self._update_plot()
 
     def NewField(self, variable, value, strong):
-        '''Slot for 'ValueChanged' signal of :py:class:`Vfield <artview.core.core.Variable>`.
+        '''
+        Slot for 'ValueChanged' signal of
+        :py:class:`Vfield <artview.core.core.Variable>`.
 
         This will:
 
-        * Reset limits
+        * Reset colormap
         * Reset units
         * Update fields MenuBox
         * If strong update: update plot
         '''
-        self._set_default_limits()
+        self._set_default_cmap(strong=False)
         self.units = None
         idx = self.fieldBox.findText(value)
         self.fieldBox.setCurrentIndex(idx)
```

Comment:
because limits is basic the zoom, it doesn't have to change, just the colorbar (with vmin and vmax) changes